apps/admin-gui/src/app/shared/components/resources-tags-list/resources-tags-list.component.html

```diff
@@ -55,8 +55,10 @@
           </div>
           <div *ngIf="isChanging.isSelected(tag)">
             <mat-form-field>
-              <mat-label>{{'VO_DETAIL.RESOURCES.TAGS.CHANGE_TAG_NAME' | translate}}</mat-label>
-              <input [(ngModel)]="tag.tagName" matInput />
+              <input
+                [(ngModel)]="tag.tagName"
+                matInput
+                placeholder="{{'VO_DETAIL.RESOURCES.TAGS.CHANGE_TAG_NAME' | translate}}" />
```

Comment:
I am not sure but to my understanding the `<mat-label>` should be used instead of placeholder value ?

Reply:
if I used mat-label this would be the result
![image](https://user-images.githubusercontent.com/47856862/221142397-57e60d7b-9f66-4e5e-b4d2-8a04fff6effa.png)

Reply:
It's true, according to mdc-migration guide: 
`The removed "legacy" appearance promoted input placeholders to the floating label if the label was not specified. All newer appearance settings require explicitly specifying a <mat-label> if one was not provided before. This change addresses an accessibility best practice of not using labels and placeholders interchangeably.`
On the other hand, our accessibility probably is not very good across all GUI, so if this is the only place where we use floating label in a table, can we maybe skip it?

But at least I would change `<mat-form-field>` to `<mat-form-field subscriptSizing="dynamic">` for better UX.